app/cmd/controller.go

```diff
@@ -161,6 +166,8 @@ func startController(c *cli.Context) error {
 		}
 	}
 
+	rebuildSyncConcurrentLimit := c.Int("rebuild-sync-concurrent-limit")
```

Comment:
Should we error out if the value is not positive?

Reply:
Theoretically, the upper caller (longhorn-manager) is responsible for guaranteeing this. But YES we can add validation separately for the engine